Add Several Parameters (#187)
DavidXanatos/sandboxie-docs: docs/Content/HideNonSystemProcesses.md

# Hide Non System Processes

HideNonSystemProcesses is a sandbox setting in [Sandboxie Ini](SandboxieIni.md).

```
   .
   .
   .
   [DefaultBox]
   HideNonSystemProcesses=y
```

Use the 'HideNonSystemProcesses=y' option to not allow sandboxed processed see processes running outside any boxes.

> Related Sandboxie Plus setting: Sandbox Options > Advanced Options > Privacy > Hide Non System Processes
> Related [Sandboxie Ini](SandboxieIni.md) setting: [HideHostProcess](HideHostProcess.md), [HideSbieProcesses](HideSbieProcesses.md)

Reviewer: ```suggestion
Related Sandboxie Plus setting: Sandbox Options > Advanced Options > Privacy > Hide Non System Processes

Related [Sandboxie Ini](SandboxieIni.md) setting: [HideHostProcess](HideHostProcess.md), [HideSbieProcesses](HideSbieProcesses.md)
```

# Hide Non System Processes

HideNonSystemProcesses is a sandbox setting in [Sandboxie Ini](SandboxieIni.md).

```
   .
   .
   .
   [DefaultBox]
   HideNonSystemProcesses=y
```

> Use the 'HideNonSystemProcesses=y' option to not allow sandboxed processed see processes running outside any boxes.

Reviewer: ```suggestion
Use the 'HideNonSystemProcesses=y' option to prevent sandboxed processed from seeing processes running outside the sandbox.
```

Author: <img width="716" alt="image" src="https://github.com/user-attachments/assets/73152a5f-c51c-43ff-bd8f-f396d73c5c96">

with `HideSbieProcesses=y` and `HideOtherBoxes=n` the process in one box can see process in another box, so maybe not "outside *the* sandbox"?

Author: Also the text in GUI says "outside any boxes"

Reviewer: https://github.com/sandboxie-plus/sandboxie-docs/pull/187#discussion_r1765414989

@DavidXanatos
